examples/test.cpp

```diff
@@ -156,9 +156,31 @@ static void test_adaptive_walnuts(const F& target_logp_grad,
   global_start_timer();
   nuts::AdaptiveWalnuts walnuts(rng, target_logp_grad, theta_init, mass_cfg,
                                 step_cfg, walnuts_cfg);
+
+  std::ofstream out("adapt_mass.csv");
```

Comment:
I feel like these changes to the example shouldn’t be committed to the main branch necessarily